apps/emqx_management/test/emqx_mgmt_api_test_util.erl

```diff
@@ -94,10 +94,14 @@ do_request_api(Method, Request, Opts) ->
     case httpc:request(Method, Request, [], []) of
         {error, socket_closed_remotely} ->
             {error, socket_closed_remotely};
-        {ok, {{"HTTP/1.1", Code, _}, _, Return}} when
+        {ok, {{"HTTP/1.1", Code, _} = Reason, Headers, Body}} when
+            Code >= 200 andalso Code =< 299 andalso ReturnBody
```

Comment:
I think we should rather have a new option like `return_status_headers_and_body` or just `return_all` or `return_full_response` or sth because `return_body` is misleading. For `2XX` we always return `Body`.

Reply:
Fixed, renamed it to `return_all`.